1-controllers/articles-controller.js

```diff
@@ -20,3 +21,15 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchArticleVotes = (req, res, next) => {
+    const {article_id} = req.params
+    const {inc_votes} = req.body
+
+    updateArticleVotes(article_id, inc_votes)
+    .then(() => selectArticleById(article_id))
+    .then((article) => {
+        res.status(200).send({article})
+    })
+    .catch(next)
+}
```

Comment:
Your second function where you selectArticleById isn't necessary - remember that we can return the results of an update or delete ect by adding a RETURNING * to the end of the sql query!